lib/routes/cdn.js

```diff
@@ -99,6 +100,19 @@ module.exports =
           const wrhsAssets = await fastify.tarballToAssets(req.raw);
           const { files, metadata: metas } = wrhsAssets;
           await fastify.uploadAssetFilesToStorage(files, expTimestamp);
+
+          if (fastify.log.security) {
+            fastify.log.security({
+              success: true,
+              message: 'Files sucessfully uploaded',
+              method: req.method,
+              url: req.url,
```

Comment:
Will the method + url have enough information to know what changed? is there additional data we might need? do we want to know what files were uploaded here?

Reply:
I've increased details at message level.